test/typings/should_not_compile/index.ts

```diff
@@ -0,0 +1,26 @@
+import {createSelector} from '../../../src/reselect.d.ts';
+import * as common from '../common';
+
+
+const implicitlyTypedSelector = createSelector(
```

Comment:
If you can't get them to work with the CI, just leave them out. But keep the tests so they can be run manually. Better than nothing, I think.